src/main/java/com/minecolonies/core/blocks/MinecoloniesCropBlock.java

```diff
@@ -126,7 +121,10 @@ public void attemptGrow(BlockState state, ServerLevel level, BlockPos pos)
     @Override
     public boolean canSurvive(@NotNull BlockState state, LevelReader level, @NotNull BlockPos pos)
     {
-        return (level.getRawBrightness(pos, 0) >= 8 || level.canSeeSky(pos)) && super.canSurvive(state, level, pos) && level.getBlockState(pos.below()).getBlock() == preferredFarmland && (preferredBiome == null || level.getBiome(pos).is(preferredBiome));
+        return (level.getRawBrightness(pos, 0) >= 8 || level.canSeeSky(pos)) 
+               && super.canSurvive(state, level, pos) 
+               && (!MineColonies.getConfig().getServer().limitCropsToFarmers.get() || level.getBlockState(pos.below()).getBlock() == preferredFarmland) 
```

Comment:
Why has this config option been introduced here? The config option states it's used to check if the crops may be planted by farmers only. This doesn't seem to be related to that.

Reply:
If the player is supposed to be able to place them, they would expect them to be placeable on normal farmland, otherwise they would be effectively gated by a farmer as otherwise they wouldn't be able to get the Minecolonies farmland otherwise.

I can change this into a separate config option though, if desired.

Reply:
Then I'd probably omit this completely, I see no reason players have to be able to cultivate these crops themselves (nobody is going to be able to do anything with them anyway as the recipes are crafter locked anyway)